molgenis-data-import/src/main/java/org/molgenis/data/importer/emx/EmxMetaDataParser.java

```diff
@@ -807,9 +824,7 @@ private void parseAttributesSheet(Repository<Entity> attributesRepo, Intermediat
 				AttributeType type = toEnum(emxDataType);
 				if (type == null)
 				{
-					throw new IllegalArgumentException(
-							"attributes.dataType error on line " + rowIndex + ": " + emxDataType
-									+ " unknown data type");
+					throw new UnknownAttributeTypeException(emxDataType, attr, rowIndex);
```

Comment:
Can we perhaps the underlying cause in an EMXException that knows the rowIndex?

Reply:
assign

Reply:
assign